alkira/resource_alkira_service_pan.go

```diff
@@ -105,10 +116,41 @@ func resourceAlkiraServicePan() *schema.Resource {
 							Type:        schema.TypeInt,
 							Computed:    true,
 						},
+						"auth_key": {
+							Description: "PAN instance auth key. This is only required " +
+								"when `panorama_enabled` is set to `true`.",
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"auth_code": {
+							Description: "PAN instance auth code. Only required when `license_type` " +
+								"is `BRING_YOUR_OWN`.",
+							Type:     schema.TypeString,
+							Optional: true,
+						},
 						"credential_id": {
 							Description: "ID of PAN instance credential managed by credential resource.",
```

Comment:
Just "ID of PAN Instance credential" should be good enough. No credential resource anymore.